docs/source/draft/design.rst

```diff
@@ -0,0 +1,56 @@
+.. _draft_design:
+
+Draft Design
+============
+
+The following document outlines the design of IAM in general terms.
+
+.. contents::
+
+
+Ownership
+---------
+
+There is a hierarchy of ownership which IAM is built around.
+
+* The OSL has Clients (Drupal.org, Apache, etc)
+* Clients have Projects (Debian mirroring, Softare builds, etc)
+* Projects have Resources (Node, Database, FTP, etc)
+* Resources have Measurements (CPU, RAM, Bandwidth, etc)
+
+Data Collection
+---------------
+
+This is the general procedure for collecting data:
+
+#. A job scheduler provokes IAM to collect data from all nodes every 30
+   minutes.
+
+    This can be provoked by Chef, Cron, whichever.
+
+#. A list of all resources is collected with their corresponding measurements
+   (CPU, Memory, Bandwidth, etc).
+
+#. For each measurement corresponding to each resource, that resource's plugin
```

Comment:
The measurement plugin, I think - resources don't have plugins, only measurements